packages/cli/src/lib/pwaValidationHelper.ts

```diff
@@ -1,31 +1,46 @@
-import {PwaValidationResult} from '@bubblewrap/validator';
-import {red, green, bold, underline, gray} from 'colors';
+import {PwaValidationResult, ScoreResult} from '@bubblewrap/validator';
+import {red, green, bold, underline, yellow} from 'colors';
 import {Log} from '@bubblewrap/core';
 
+function getColor(score: ScoreResult): string {
+  switch (score.status) {
+    case 'PASS': return green(score.printValue);
+    case 'WARN': return yellow(score.printValue);
+    case 'FAIL': return red(score.printValue);
+    default: return score.printValue;
+  }
+}
+
 export function printValidationResult(validationResult: PwaValidationResult, log: Log): void {
   log.info('');
   log.info('Check the full PageSpeed Insights report at:');
   log.info(`- ${validationResult.psiWebUrl}`);
   log.info('');
 
-  const performanceValue = validationResult.scores.performance.status === 'PASS' ?
-  green(validationResult.scores.performance.printValue) :
-  red(validationResult.scores.performance.printValue);
-
-  const pwaValue = validationResult.scores.pwa.status === 'PASS' ?
-  green(validationResult.scores.pwa.printValue) :
-  red(validationResult.scores.pwa.printValue);
+  const performanceValue = getColor(validationResult.scores.performance);
+  const pwaValue = getColor(validationResult.scores.pwa);
 
   const overallStatus = validationResult.status === 'PASS' ?
-  green(validationResult.status) : red(validationResult.status);
+      green(validationResult.status) : red(validationResult.status);
 
-  const accessibilityValue = gray(validationResult.scores.accessibility.printValue);
+  const accessibilityValue = validationResult.scores.accessibility.printValue;
+
+  const fcpValue = getColor(validationResult.scores.firstContentfulPaint);
+  const lcpValue = getColor(validationResult.scores.largestContentfulPaint);
+  const fidValue = getColor(validationResult.scores.firstInputDelay);
+  const clsValue = getColor(validationResult.scores.cumulativeLayoutShift);
 
   log.info('');
   log.info(underline('Quality Criteria scores'));
   log.info(`Lighthouse Performance score: ......... ${performanceValue}`);
   log.info(`Lighthouse PWA check: ................. ${pwaValue}`);
   log.info('');
+  log.info(underline('Web Vitals'));
+  log.info(`First Contentful Paint (FCP) .......... ${fcpValue}`);
+  log.info(`Largest Contentful Paint (LCP) ........ ${lcpValue}`);
+  log.info(`First Input Delay (FID) ............... ${fidValue}`);
```

Comment:
Technically isn't this Total Blocking Time, not First Input Delay?

https://web.dev/vitals/#lab-tools-to-measure-core-web-vitals

Reply:
Lighthouse calls this `maxPotentialFID`. Maybe rename this to "Maximum Potential First Input Delay (Max FID)"?

Reply:
Yeah, that sounds good - hope it fits though.

Reply:
Done. it fits - we're around 65 columns with that.

![Screenshot 2020-06-03 at 10 26 04](https://user-images.githubusercontent.com/1733592/83620039-a9210100-a584-11ea-8730-550dcdf9cd97.png)